packages/Bio/src/utils/multiple-sequence-alignment-ui.ts

```diff
@@ -0,0 +1,81 @@
+import * as grok from 'datagrok-api/grok';
+import * as DG from 'datagrok-api/dg';
+import * as ui from 'datagrok-api/ui';
+
+import {ALPHABET, NOTATION} from '@datagrok-libraries/bio/src/utils/macromolecule';
+import {runKalign} from './multiple-sequence-alignment';
+import {pepseaMethods, runPepsea} from './pepsea';
+import {checkInputColumnUI} from './check-input-column';
+
+export function multipleSequenceAlignmentUI(col: DG.Column<string> | null = null) {
+  const table = col?.dataFrame ?? grok.shell.t;
+  const seqCol = col ?? table.columns.bySemType(DG.SEMTYPE.MACROMOLECULE);
+  if (seqCol == null) {
+    grok.shell.warning(`MSAError: dataset doesn't conain any Macromolecule column`);
+    return;
+  }
+
+  let performAlignment: () => Promise<DG.Column<string> | null> = async () => null;
+  const methodInput = ui.choiceInput('Method', pepseaMethods[0], pepseaMethods);
+  methodInput.setTooltip('Alignment method');
+  const gapOpenInput = ui.floatInput('Gap open', 1.53);
+  gapOpenInput.setTooltip('Gap opening penalty at group-to-group alignment');
+  const gapExtendInput = ui.floatInput('Gap extend', 0);
+  gapExtendInput.setTooltip('Gap extension penalty to skip the alignment');
+  const inputRootStyles = [methodInput.root.style, gapOpenInput.root.style, gapExtendInput.root.style];
+
+  const colInput = ui.columnInput('Sequence', table, seqCol, () => {
+    const potentialCol = colInput.value;
+    const unusedName = table.columns.getUnusedName(`msa(${potentialCol.name})`);
+
+    if (checkInputColumnUI(
+      potentialCol, potentialCol.name, [NOTATION.FASTA], [ALPHABET.DNA, ALPHABET.RNA, ALPHABET.PT], false)) {
+      for (const inputRootStyle of inputRootStyles)
+        inputRootStyle.display = 'none';
```

Comment:
maybe set the class on these inputs where you will set the display to none?

Reply:
class div.ui-form.inputs-hidden input[field-set=1] with display: none;

field-set - attribute of an input element

set form div class with 'inputs-hidden'